sdcm/scylla_bench_thread.py

```diff
@@ -191,14 +191,14 @@ def _parse_bench_summary(lines):  # pylint: disable=too-many-branches
                     line.startswith('Rows per request:') or line.startswith('Page size:') or \
                     line.startswith('Concurrency:') or line.startswith('Connections:') or \
                     line.startswith('Maximum rate:') or line.startswith('Client compression:'):
-                split_idx = line.index(':')
-                key = line[:split_idx].strip()
-                value = line[split_idx + 1:].split()[0]
+                split = line.split(':', maxsplit=1)
```

Comment:
please add an example of the line do we can understand the parsing

Reply:
done